src/Peachpie.CodeAnalysis/Semantics/Graph/BuilderVisitor.cs

```diff
@@ -259,18 +259,20 @@ private void AddFinalReturn()
             return block;
         }
 
-        private CatchBlock/*!*/NewBlock(CatchItem item)
+        private BoundBlock/*!*/NewUnorderedBlock() => new BoundBlock();
```

Comment:
just rename it back please

Reply:
The original `NewBlock()` is still there (and still used on some places, e.g. passed as a delegate to a generator builder), so it would collide with it. I wanted to distinguish this method from it as it doesn't allocate the `Ordinal` number (and the created block is expected to obtain it otherwise).

Reply:
I reverted these changes in `BuilderVisitor` in bd78e6ab60247eb0a3ca96f0157d225d8334139c.